Integrations/HelloWorld/HelloWorld_test.py

```diff
@@ -0,0 +1,24 @@
+from HelloWorld import Client, say_hello_command, say_hello_over_http_command
```

Comment:
what about a test for the fetch-incidents?

Reply:
I was thinking to show that too, but wondered whether it will be too complicated or not. I may add that test.
Or just refer to ProofpointTAP example